atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownGetTest.java

```diff
@@ -112,14 +112,14 @@ public void getRangeOfTimestampsThrows() {
                 OneNodeDownTestSuite.TEST_TABLE, range, Long.MAX_VALUE);
         assertThatThrownBy(() -> it.next())
                 .isExactlyInstanceOf(InsufficientConsistencyException.class)
-                .hasMessage("This operation requires all Cassandra nodes to be up and available.");
+                .hasMessageContaining("requires all Cassandra nodes to be up and available.");
     }
 
     @Test
     public void getAllTimestampsThrows() {
         assertThatThrownBy(() -> OneNodeDownTestSuite.kvs.getAllTimestamps(OneNodeDownTestSuite.TEST_TABLE,
                 ImmutableSet.of(OneNodeDownTestSuite.CELL_1_1), Long.MAX_VALUE))
                 .isExactlyInstanceOf(InsufficientConsistencyException.class)
-                .hasMessage("This get operation requires ALL Cassandra nodes to be up and available.");
+                .hasMessageContaining("requires ALL Cassandra nodes to be up and available.");
```

Comment:
Our insufficient consistency exception messages are not sufficiently consistent! ("all" vs "ALL")